src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -204,7 +205,7 @@ class ActivityFeed extends React.Component<Props, State> {
                             'bcs-is-disabled': isDisabled,
                         })}
                         createComment={hasCommentPermission ? this.onCommentCreate : noop}
-                        createTask={hasCommentPermission ? this.onTaskCreate : noop}
+                        createTask={hasCommentPermission && onTaskCreate ? this.onTaskCreate : null}
```

Comment:
don't need this?

Reply:
Yeah, it looks like `onTaskCreate` is defaulted to `noop` in `this.taskCreate`.

Reply:
That was lingering from passing down props. Will fix, thanks for catching!